tests/bluetooth/tester/src/l2cap.c

```diff
@@ -138,7 +138,7 @@ static void disconnected_cb(struct bt_l2cap_chan *l2cap_chan)
 	tester_send(BTP_SERVICE_ID_L2CAP, L2CAP_EV_DISCONNECTED,
 		    CONTROLLER_INDEX, (uint8_t *) &ev, sizeof(ev));
 }
-
+#if defined(CONFIG_BT_L2CAP_ECRED)
```

Comment:
Please leave an empty line between the functions, even though you're adding the ifdef